app/views/collection_drafts/forms/fields/_geometry_fields.html.erb

```diff
@@ -48,74 +48,58 @@
   <% g_polygons_checked = !geometry['GPolygons'].nil? %>
```

Comment:
trying to add a comment to line 30:
- the required icon is not showing up for Coordinate System for me, so I was showing an error on the preview circles (for Spatial Extent), and when I clicked on it, no validation prompt was telling me what was missing on the form
- When I click on the `clear` link under Coordinate System, any geometry information I have typed into the text box inputs (points, bounding rectangles, etc) gets cleared out as well (but not the checkboxes). Not sure if that is the desired behavior